ui_release.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e
+
+cd ui/
+
+files=("../LICENSE" "../CHANGELOG" ".npmignore")
+workspaces=$(npm ls --production --depth 1 -json | jq -r '.dependencies[].resolved[11:]')
+
+function copy() {
+  for file in "${files[@]}"; do
+    for workspace in ${workspaces}; do
+      if [ -f "${file}" ]; then
+        cp "${file}" "${workspace}"/"$(basename "${file}")"
+      fi
+    done
+  done
+}
+
+function publish() {
+  npm publish --workspaces
```

Comment:
I guess this one won't work since the `@perses-dev/app` is a private package. So perhaps we should manually loop other workspace and skip the app

Reply:
It's fixed now